demo/src/test/java/com/mongodb/jdbc/utils/TestUtils.java

```diff
@@ -0,0 +1,439 @@
+package com.mongodb.jdbc.utils;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public abstract class TestUtils {
+
+    protected Connection m_conn;
+    protected DatabaseMetaData m_dbMeta;
+
+    protected abstract Connection connect() throws Exception;
+
+    @BeforeEach
+    private void execConnection() throws Exception {
+        m_conn = connect();
+        m_dbMeta = m_conn.getMetaData();
+    }
+
+    @AfterEach
+    private void execDeconnection() throws Exception {
+        m_conn.close();
+    }
+
+    @Test
+    void testConnection() throws Exception {
+        System.out.println(m_dbMeta.getDriverName());
+        System.out.println(m_dbMeta.getDriverVersion());
+
+        System.out.println("Connected !");
+    }
+
+    @Test
+    void testExecute() throws Exception
+    {
```

Comment:
This may be addressed by running the formatter, but it looks like our braces are inconsistent in this file/PR. In the other files, the braces are in-line (i.e. ` public void method() {`), but for this method and most that follow, the braces are on their own lines (i.e.
```
public void method()
{
```
). Can we stick with the former convention for the sake of stylistic consistency? Ideally, just running the formatter will do this for us by enforcing one convention over the other. I'm indifferent which direction it goes, but think consistency is useful.